refiner/app/services/terminology.py

```diff
@@ -66,16 +66,26 @@ def from_grouper_row(cls, row: GrouperRow) -> "ProcessedGrouper":
             codes=all_codes,
         )
 
-    def build_xpath(self, search_in: str = "observation") -> str:
+    def build_xpath(self, search_in: str = "any") -> str:
```

Comment:
Since it seems like we only want `any` or `observation` do you think it'd make sense to reflect that in our typing?

```suggestion
    def build_xpath(self, search_in: Literal["any", "observation"] = "any") -> str:
```


We get nicer type hints as a bonus 😃 
<img width="560" alt="image" src="https://github.com/user-attachments/assets/1480b078-3f33-42e1-81e1-f6315ee65462" />

Reply:
oh yeah, that's a good point. I meant to ask that as well but didn't 🙃 if there's no need for it then I think this is a good path forward.